.github/workflows/pr-cleanup.yaml

```diff
@@ -0,0 +1,29 @@
+name: Clean up npm pr tags on release
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  release-pr:
+    if: github.event.pull_request.merged == true
```

Comment:
I think it should clean if it's pull request is not merged

Reply:
Yes, I agree. The NPM tag should be removed regardless of whether the PR was merged.

Reply:
this means that we are just going to remove this if